compute/cmdline/src/main/java/ComputeEngineSample.java

```diff
@@ -157,17 +160,22 @@ public static boolean printInstances(Compute compute) throws IOException {
   public static Operation startInstance(Compute compute, String instanceName) throws IOException {
     System.out.println("================== Starting New Instance ==================");
 
-
     // Create VM Instance object with the required properties.
     Instance instance = new Instance();
     instance.setName(instanceName);
     instance.setMachineType(
         "https://www.googleapis.com/compute/v1/projects/"
-        + PROJECT_ID + "/zones/" + ZONE_NAME + "/machineTypes/n1-standard-1");
+            + PROJECT_ID
+            + "/zones/"
+            + ZONE_NAME
+            + "/machineTypes/n1-standard-1");
```

Comment:
Perhaps we could create this string w/ `String.format(...)`   Also, we should probably use `e2-standard-1` as well.